README.md

```diff
@@ -4,8 +4,7 @@ Tools for doing x way meta-analysis
 
 ## Variant matching across studies
 
-Variants are matched using chr pos ref and alt. For this reason all 37 build results need to first be lifted over to 38.
-[lift.wdl](wdl/lift.wdl) can be used to liftover results first if needed.
+Variants are matched using chr pos ref and alt. For this reason all 37 build results need to first be lifted over to 38 to match FinnGen. [lift.wdl](wdl/lift.wdl) can be used to liftover results first if needed.
```

Comment:
can you also add section for how to phenotype QC external sumstats i.e. point to that script and give column explanations